### PR TITLE
bump uuid

### PR DESCRIPTION
**Pull request by kgc00**
# Why
> warning eas-cli > @expo/rudder-sdk-node > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

# How
Bumped UUID

# Test Plan
Tested client, confirmed events made it to backend.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10242,11 +10242,11 @@ utils-merge@1.0.1:
   resolved "https://registry.yarnpkg.com/utils-merge/-/utils-merge-1.0.1.tgz#9f95710f50a267947b2ccc124741c1028427e713"
   integrity sha1-n5VxD1CiZ5R7LMwSR0HBAoQn5xM=
 
-uuid@^3.0.1, uuid@^3.2.1, uuid@^3.3.2:
+uuid@^3.0.1, uuid@^3.3.2:
   version "3.3.2"
   resolved "https://registry.yarnpkg.com/uuid/-/uuid-3.3.2.tgz#1b4af4955eb3077c501c23872fc6513811587131"
```
Comment (ide) [inline yarn.lock:10247]: If this is required by a prod dependency we'll want to remove it

**Review by ide — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (kgc00) [reply yarn.lock:9974]: it's from nyc, snyk, & codecov (all of which are dev-dependencies). should i take any action?

Comment (ide) [reply yarn.lock:9974]: No, that's fine.

Comment (kgc00) [reply yarn.lock:9974]: thanks!

**Pull request merged**